src/Corax/IndexSearcher.cs

```diff
@@ -31,12 +32,12 @@ public sealed unsafe class IndexSearcher : IDisposable
 
         public bool IsAccelerated => Avx2.IsSupported && !ForceNonAccelerated;
 
+        private readonly bool _ownsTransaction = true;
```

Comment:
Let's not set it to `true` by default. Better to set it explicitly in the relevant constructor so we can easily find where the searcher opens the transaction on his own.